src/components/projects/CreateProjectForm.vue

```diff
@@ -740,31 +619,88 @@
             Astronomical Dark & Moon Alt &lt; 6
           </b-checkbox>
         </b-field>
-        <b-field
-          label="Generic Instrument"
-          style="margin-top: 1em;"
+        <div
+          class="flex-row"
+          style="margin-top: 1em; gap: 3em;"
         >
-          <b-select v-model="generic_instrument">
-            <option value="Main Camera">
-              Main Camera
-            </option>
-            <option value="Auxiliary Camera">
-              Auxiliary Camera
-            </option>
-            <option value="Echelle Spectrometer">
-              Echelle Spectrometer
-            </option>
-            <option value="UXEX Spectrometer">
-              UXEX Spectrometer
-            </option>
-            <option value="Planet Camera">
-              Planet Camera
-            </option>
-            <option value="IR Photometer">
-              IR Photometer
-            </option>
-          </b-select>
-        </b-field>
+          <b-field label="Defocus (mags)">
```

Comment:
can you make the changes to the code without deleting everything? it's difficult to go back and forth and understand what the changes were and for what. In other words, can you leave the code as it was and can you change the values and modify it based on the original code? Maybe I'm misreading it, but it seems like this big chunk does the same functionality as lines 430-518 (except for a few values). Please correct me if I'm wrong :-)

Reply:
and one more request for best practice and easing the review process, can you make commits a bit more clear and maybe more frequently? this may be a personal request and it's okay if the answer is no. Maybe we should talk to Mark but I think it's best practice to commit more and be descriptive

Reply:
Oh I think you moved this chunk of code because it has to be elsewhere? Is that correct?

Reply:
Correct, moving the additional details panel to the advanced tab

Reply:
alrighty! I think this looks good, then :-)